docs/proposals/rename-group.md

```diff
@@ -0,0 +1,156 @@
+- Change the `topolvm.cybozu.com/pvc` finalizer to` topolvm.io/pvc`
+
+##### PersistentVolume
+
+Modify each PersistentVolume resource in TopoLVM as follows:
+
+- Change the value of the `pv.kubernetes.io/provisioned-by` annotation to` topolvm.io`
+- Change the value of the `.spec.csi.driver` field to` topolvm.io`
+- Change the string `topolvm.cybozu.com` in the value of the `.spec.csi.volumeAttributes ["storage.kubernetes.io/csiProvisionerIdentity "]` field to` topolvm.io`
+
+**Since it is not possible to change the above fields of the PersistentVolume resource, it is necessary to recreate the resource when migrating the group name. Also, if `.spec.persistentVolumeReclaimPolicy = Delete` is set, there is a risk of deleting the actual LVM volume, so it is recommended to temporarily change it to` Retain` and then delete it.**
+
+### Enable the use of topolvm.cybozu.com
+
+We will make the following changes to allow users to continue to use `topolvm.cybozu.com`.
+
+#### Addition of group name setting function
+
+- If you set the `USE_LEGACY_PLUGIN_NAME` environment variable, make sure that the group name uses` topolvm.cybozu.com`
+- If the `USE_LEGACY_PLUGIN_NAME` environment variable is not set, the group name` topolvm.io` will be used
+
+#### Group name setting in helm chart
+
+Add the `.Values.useLegacyName` variable to the helm chart and set this variable to` true` to set the installation manifest to use `topolvm.cybozu.com`.
+If the `.Values.useLegacyName` variable is not set to` true`, the group name will be `topolvm.io`.
+
+If you have set the following variables, you may need to set them appropriately to use `topolvm.io` or` topolvm.cybozu.com` in the variables.
+
+- `.Values.storageClasses`
+- `.Values.node.volumes`
+- `.Values.volumeMounts.topolvmNode`
+
+#### Automatic generation for API and CRD
+
+Automatically generating the API and CRD of `topolvm.cybozu.com` based on the API and CRD of` topolvm.io`.
+
+### Things to be done by users of topolvm.cybozu.com for upgrading the helm chart
+
+Since this change was released, users of TopoLVM with the group name `topolvm.cybozu.com` will need to take the following actions when upgrading the helm chart.
+
+- Set the `.Values.useLegacyName` variable to` true` in `values.yaml` in the helm chart
+- If there is a part of the `values.yaml` in the helm chart that uses the string `topolvm.cybozu.com`, change it to` topolvm.io`
+
+If you upgrade the helm chart without taking above actions, it is possible that TopoLVM related resources will be deleted.
+
+### Migrate from topolvm.cybozu.com to topolvm.io
+
+If you are already using TopoLVM with the group name `topolvm.cybozu.com` and want to migrate the group name to` topolvm.io` after the release of this change, you can manually change the data by the following method.
+
+1. Avoid booting pods using TopoLVM volumes during migration
+1. Temporarily stop the following pods:
+  - topolvm-controller
+  - topolvm-node
+  - topolvm-scheduler
```

Comment:
```suggestion
   - topolvm-controller
   - topolvm-node
   - topolvm-scheduler
```
Currently, numbering restarts from 1 at line 143. Changes above will prevent it.